nntrainer/layers/tflite_layer.cpp

```diff
@@ -102,32 +102,25 @@ void TfLiteLayer::forwarding() {
   if (net_input.size() != input_dim.size())
     throw std::invalid_argument("Provided number of input dimensions mismatch");
 
-  for (int idx = 0; idx < dims.size(); idx++) {
+  for (unsigned int idx = 0; idx < dims.size(); idx++) {
     if (net_input[idx]->getDim() != input_dim[idx])
       throw std::invalid_argument("Input dimensions mismatch");
   }
 #endif
-
-  sharedConstTensors out;
-
   auto in_indices = interpreter->inputs();
   for (size_t idx = 0; idx < net_input.size(); idx++)
     interpreter->tensor(in_indices[idx])->data.raw =
       (char *)net_input[idx]->getVariableRef().getData();
 
   auto out_indices = interpreter->outputs();
-  out.resize(out_indices.size());
   for (size_t idx = 0; idx < out_indices.size(); idx++) {
-    out[idx] = MAKE_SHARED_TENSOR(output_dim[idx]);
     interpreter->tensor(out_indices[idx])->data.raw =
-      (char *)out[idx]->getData();
+      reinterpret_cast<char *>(net_hidden[idx]->getVariableRef().getData());
```

Comment:
I think we need to check the dimension.

Reply:
It will match as line:60 sets dimension from tflite, I added a failsafe checker for DEBUG mode :)